src/humanize/number.py

```diff
@@ -26,6 +26,17 @@
 NumberOrString: TypeAlias = "float | str"
 
 
+def format_non_finite(value: float) -> str:
```

Comment:
Let's have this as a "private" function:
```suggestion
def _format_non_finite(value: float) -> str:
```

Reply:
I'm making it private...will it be merged then?

Reply:
Will merge when it's ready, thank you for all your work!

Reply:
What more to add?